docs: document how to publish a new version on npm (#3)
fabrictech/dynamodb-localhost: README.md

Internally, we've integrated ESLint and a pretty vanilla airbnb config, which led to a general refactor. Most notably, we've introduced es6 where applicable.

### Publishing New Versions

If you make changes to this package, they'll need to be published to `npm` in order to be available to dependents, such as `linen`.

Please take the following steps:

> 1. Bump the package version, in package.json, as appropriate. For ex, for a minor version bump, `0.3.2 -> 0.3.3`

Reviewer: Nit: I'd recommend using `npm version` for this:

```suggestion
1. Bump the package version using [`npm version`](https://docs.npmjs.com/cli/v7/commands/npm-version). For ex, for a patch version bump, run `npm version patch`.
```
